pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 name = "notte"
-version = "1.3.3"
+version = "1.3.5"
```

Comment:
Can you do this for all subpyprojects or not do it at all ? They need to be in sync

Reply:
(FYI I added an exception at package load time if notte_browser, notte_core, ... do not share the same version)